src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const liArray = [...document.querySelectorAll('li')];
+const liEl = liArray.filter((li) => li.querySelector('ul'));
+
+liEl.forEach((li) => {
+  const span = document.createElement('span');
+
+  span.textContent = li.firstChild.textContent;
+  li.firstChild.remove();
+  li.prepend(span);
+});
+
+tree.addEventListener('click', (e) => {
+  if (e.target.tagName === 'SPAN') {
+    const parentLi = e.target.parentElement;
+    const subList = parentLi.querySelector('ul');
+
+    if (subList) {
+      subList.style.display =
+        subList.style.display === 'none' ? 'block' : 'none';
```

Comment:
There is an issue with the assignment of 'subList.style.display'. The current code does not assign any value to 'subList.style.display' on line 21. You should assign the result of the conditional expression to 'subList.style.display'.